packages/libs/eda/src/lib/core/components/visualizations/implementations/BipartiteNetworkVisualization.tsx

```diff
@@ -64,32 +76,110 @@ interface Options
 // The bipartite network takes no input variables, because the received data will complete the plot.
 // Eventually the user will be able to control the significance and correlation coefficient values.
 function BipartiteNetworkViz(props: VisualizationProps<Options>) {
-  const { options, updateThumbnail } = props;
+  const {
+    options,
+    computation,
+    visualization,
+    updateThumbnail,
+    computeJobStatus,
+    filteredCounts,
+    filters,
+  } = props;
+
+  const studyMetadata = useStudyMetadata();
+  const { id: studyId } = studyMetadata;
+  const entities = useStudyEntities(filters);
+  const dataClient: DataClient = useDataClient();
+  const computationConfiguration: CorrelationAssayMetadataConfig = computation
+    .descriptor.configuration as CorrelationAssayMetadataConfig;
 
   // Fake data
-  const data: BipartiteNetworkData = useMemo(
-    () => genBipartiteNetwork(100, 10),
-    []
+  const data = usePromise(
+    useCallback(async (): Promise<BipartiteNetworkResponse | undefined> => {
+      // Only need to check compute job status and filter status, since there are no
+      // viz input variables.
+      console.log(filteredCounts);
+      if (computeJobStatus !== 'complete') return undefined;
+      if (filteredCounts.pending || filteredCounts.value == null)
+        return undefined;
+
+      // There are _no_ viz request params for the volcano plot (config: {}).
+      // The data service streams the volcano data directly from the compute service.
```

Comment:
another reference to volcano here. this time though i think my larger point is that for the network i think we said itd end up being the data service job to do the pruning of edges by pvalue and correlation coefficient thresholds. (i dont actually remember the compute or data service prs well enough to remember if thats what we actually ended up doing. but think we should. it was a good plan.)

Reply:
anyway, just trying to say there will be viz plugin params, eventually, if not yet.